waywiser_agrarsense/config/agrarsense.yaml

```diff
@@ -0,0 +1,26 @@
+/**:
+  # general parameters shared by all agrarsense nodes
+  ros__parameters:
+    ros_async: true
+    ros_host_ip: "127.0.0.1"
+    ros_port: 9090
+    map_name: "playground" # possible options: playground, vindeln, rovaniemiforest
+    ego_vehicle_identifier: "forwarder"
+
+sim_orchestrator_node:
+  ros__parameters:
+    sim_script_path: "~/workspaces/agrarsense_ws/Agrarsense.sh"
+    ros_bridge_script_path: "agrarsense_ros_bridge.sh"
+    sigterm_timeout: 20.0
+    sim_in_command_topic: "/agrarsense/in/commands"
+    workersthreadpool: 0 #By default the Simulator uses maximum number of CPU threads - 1
+    spectator_rendering: true
+    quality_level: "Balanced" #"Ultra", "Balanced" or "Low"
+    # spectator_transform: "85,2650,1200,0,270,0"
+    sim_startup_time: 5.0
+    map_loading_time: 2.0
+    command_publish_delay: 2.0
+    objects_json_path: "playground_objects.json"
```

Comment:
Will this only work with the "playground" map or any map? I think it would be good to point to the respective part in the AGRARSENSE simulator documentation.

Reply:
The objects in the JSON file would spawn in any map. I just named the file to indicate that the spawn locations for the objects are set for the playground map.